Insertion_sort.c

```diff
@@ -0,0 +1,35 @@
+#include "sort.h"
+
+/**
+ * insertion_sort_list - perform insertion sort on a linked list
+ * @list: head node of the linked list
+ */
```

Comment:
The main comment at the beginning of the code provides a clear and concise description of the purpose of the function `insertion_sort_list`.